### PR TITLE
Increase Test Coverage and Drop Webform Importer

### PR DESCRIPTION
**Pull request by aaron-lane**
This pull request continues to increase test coverage of the codebase, sporadically focusing on some of the least covered classes and functions.

The obsolete `Parser` and `WebformImporter` tasks are removed.

The `BaseCsvImporter` parameter `csvpath` is renamed to `csv_lines` to reflect that it actually expects an array of file lines.

A wayward `</div>` tag in `PollState.html` is removed.

The output of `make coverage` is tweaked to show, and sort by, lines missed.

### CODE REVIEW

**Comment by FLRota** [comment]
Hey Aaron, that's a lot of Commits! Thanks very much for your help!

File: app/templates/admin/index.html [hunk 1]
```diff
@@ -66,27 +66,6 @@
...
-                    </th>
-                    <th align="right">
-                        <input class="btn_upload" style="padding: 5px 15px;" type="submit" value="Import Data" />
-                    </th>
-                </form>
-            </tr>
-        </table>
-    </div>
-    {% endif %}
     {% if perms.app.can_import_third_party %}
```
Comment (bwdmonkey) [inline app/templates/admin/index.html:69]: Can we remove this as well? In this pr or later

File: app/templates/app/PollState.html [hunk 2]
```diff
@@ -43,7 +43,7 @@
 
 <p>Your task is <span id="user-count">starting</span>!</p>
 
-<div class="status"></div>
```
Comment (bwdmonkey) [inline app/templates/app/PollState.html:46]: 👀 thanks 😂

File: app/worker/tasks/test__init__.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+from app.models.donor import Donor
+from app.worker import tasks
+
+
+class InitTestCase(TestCase):
+    def setUp(self) -> None:
+
+        return super().setUp()
+
+    def test_receiptor(self) -> None:
```
Comment (bwdmonkey) [inline app/worker/tasks/test__init__.py:16]: Can we add one more test case here to check if we receipt more than one (2+) donations we get a zip file instead?

**Review by bwdmonkey — APPROVED** [review]
LGTM w suggestions 👍👍

File: app/templates/admin/index.html (hunk 1 above)
Comment (aaron-lane) [reply app/templates/admin/index.html]: Done in https://github.com/CodeTheChangeUBC/reBOOT/pull/343/commits/b82b55a0092d3176e86678ac20a034933013f7fa.

File: app/worker/tasks/test__init__.py (hunk 3 above)
Comment (aaron-lane) [reply app/worker/tasks/test__init__.py]: Added in https://github.com/CodeTheChangeUBC/reBOOT/pull/343/commits/e3e75ee1516fb0f68caa4f57d62bb7040bdb72e9.

**Pull request merged**